loader/generated/vk_layer_dispatch_table.h

```diff
@@ -275,7 +275,9 @@ typedef struct VkLayerInstanceDispatchTable_ {
 } VkLayerInstanceDispatchTable;
 
 // Device function pointer dispatch table
+#define DEVICE_DISP_TABLE_MAGIC_NUMBER 0x10ADED040410ADEDUL
```

Comment:
We already validate the instance level object in every place.  It was just duplicate checking so I removed it to be more optimal.  I don't have any other mechanism of validating the device object, though, so that's why I have this.